release-please-config.json

```diff
@@ -56,6 +56,7 @@
     "plugins/node/opentelemetry-instrumentation-restify": {},
     "plugins/node/opentelemetry-instrumentation-router": {},
     "plugins/node/opentelemetry-instrumentation-winston": {},
+    "plugins/node/opentelemetry-winston-transport": {},
```

Comment:
there is also a second file `.release-please-manifest.json` that needs to be adapted. 
doing this will make the `release-please-validate` workflow pass.